emx/rest_api.py

```diff
@@ -396,3 +396,49 @@ def cancel_all(self, contract_code=None):
         self._headers['EMX-ACCESS-SIG'] = signature.decode().strip()
         self._headers['EMX-ACCESS-TIMESTAMP'] = str(timestamp)
         return self.session.delete(url=url, json=body, headers=self._headers)
+
+    @handle_result
+    def positions(self, contract_code=""):
+        """Returns positions
+        """
+
+        endpoint = "/v1/positions"
+        url = self.uri + endpoint
+
+        body = {
+            "contract_code": contract_code,
+           
+        }
+        timestamp = get_timestamp()
+        signature = generate_signature(self._api_secret, timestamp, "GET", endpoint, body)
+
+        self._headers['EMX-ACCESS-KEY'] = self._api_key
+        self._headers['EMX-ACCESS-SIG'] = signature.decode().strip()
+        self._headers['EMX-ACCESS-TIMESTAMP'] = str(timestamp)
+        return self.session.get(url=url, json=body, headers=self._headers)
+
+    @handle_result
+    def create_new_order_market(self, contract_code, order_type,
```

Comment:
@romeucampos does `create_new_order()` not already work for the market order case? should be able to pass `order_type` as `'market'` to that existing function (cc @PavelPetukhov )

Reply:
does not work, because in the market can not have price parameter, nor empty price "". You have to take the price parameter to work properly.

Reply:
Ah, gotcha. We're pushing a fix to that on our side. That'll allow the old function to work. I'll push your new `positions()` function and add some support for new order types to `create_new_order()` while we're here.